streamer/m3u8_concater.py

```diff
@@ -251,10 +242,13 @@ def _get_bandwidth(var_playlists: List['MediaPlaylist'],
       band = max(int(var_playlist.stream_info['BANDWIDTH']), band)
       avg_band += int(var_playlist.stream_info['AVERAGE-BANDWIDTH']) * duration
     
-    return str(band), str(math.ceil(avg_band/sum(durations)))
+    return {
+        'BANDWIDTH': str(band),
+        'AVERAGE-BANDWIDTH': str(math.ceil(avg_band/sum(durations)))
```

Comment:
Wow.  I'm really surprised to check the spec and find that these values must be integers.  Nice attention to detail!

I probably would have just written out a float, which would have worked fine in a JS player and may have broken some other player implementation.